### PR TITLE
PLAT-52971: Fixed lists to support RTL by dynamic language changes.

### PR DESCRIPTION
**Pull request by 0x64**
### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed lists to support RTL by dynamic language changes.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
When RTL/LRT is changed, a list will re-render items and renew the scroll position.

### Links
[//]: # (Related issues, references)
PLAT-52971

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: packages/ui/Scrollable/ScrollableNative.js [hunk 1]
```diff
@@ -389,6 +389,8 @@ class ScrollableBaseNative extends Component {
 			if (!this.deferScrollTo) {
 				this.scrollTo(this.scrollToInfo);
 			}
+		} else if (rtl !== prevState.rtl) {
```
Comment (MikyungKim) [inline packages/ui/Scrollable/ScrollableNative.js]: I think we can handle the position in VirtualListBase only.

Comment (0x64) [reply packages/ui/Scrollable/ScrollableNative.js]: I re-implemented not to change Scrollables.

File: packages/ui/VirtualList/VirtualListBase.js [hunk 2]
```diff
@@ -240,6 +240,17 @@ const VirtualListBaseFactory = (type) => {
 			} else if (this.hasDataSizeChanged) {
 				this.updateStatesAndBounds(nextProps);
 				this.setContainerSize();
+			} else if (rtl !== nextProps.rtl) {
+				const
+					{scrollPosition} = this,
+					{x, y} = this.isPrimaryDirectionVertical ? {x: 0, y: scrollPosition} : {x: scrollPosition, y: 0};
```
Comment (MikyungKim) [inline packages/ui/VirtualList/VirtualListBase.js]: I think we can make a function for getting x, y.
`composeStyle` also uses this logic.

**Comment by MikyungKim** [comment]
LGTM

**Pull request merged**